docs/install_guides/installing-openvino-from-archive-linux.md

```diff
@@ -237,41 +259,8 @@ If you have more than one OpenVINO version installed on your system, you can eas
    Next time when you open a terminal, you will see ``[setupvars.sh] OpenVINO™ environment initialized``. 
    Changing ``.bashrc`` is not recommended when you have multiple OpenVINO versions on your machine and want to switch among them.
 
-The environment variables are set. Continue to the next section if you want to download any additional components.
-
-Step 3 (Optional): Install Additional Components
-++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
-
-OpenVINO Development Tools is a set of utilities for working with OpenVINO and OpenVINO models. 
-It provides tools like model conversion API, Benchmark Tool, Post-Training Optimization Tool, and Open Model Zoo Downloader. 
-If you install OpenVINO Runtime using archive files, OpenVINO Development Tools must be installed separately.
-
-See the :doc:`Install OpenVINO Development Tools <openvino_docs_install_guides_install_dev_tools>` 
-page for step-by-step installation instructions.
-
-OpenCV is necessary to run demos from Open Model Zoo (OMZ). Some OpenVINO samples can also extend their 
-capabilities when compiled with OpenCV as a dependency. To install OpenCV for OpenVINO, see the 
-`instructions on GitHub <https://github.com/opencv/opencv/wiki/BuildOpenCV4OpenVINO>`__.
-
-Step 4 (Optional): Configure Inference on Non-CPU Devices
-++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
-
-OpenVINO Runtime has a plugin architecture that enables you to run inference on multiple devices 
-without rewriting your code. Supported devices include integrated GPUs, discrete GPUs and GNAs. 
-See the instructions below to set up OpenVINO on these devices.
+The environment variables are set.
 
-.. tab-set::
-
-   .. tab-item:: GPU
-      :sync: gpu
-   
-      To enable the toolkit components to use processor graphics (GPU) on your system, follow the steps in :ref:`GPU Setup Guide <gpu guide>`.
-   
-   .. tab-item:: GNA
-      :sync: gna
-   
-      To enable the toolkit components to use Intel® Gaussian & Neural Accelerator (GNA) on your system, follow the steps in :ref:`GNA Setup Guide <gna guide>`.
```

Comment:
In general, I want to keep most of content of install documentation as is and avoid its rewriting each release. 
Developers are users are confused when each release they read refactored pages, while actually pages were in a good state.

Reply:
These fragments add a lot of redundant text. Instead of listing all devices and giving links to particular guides, we've decided to use one wording for all distributions:

Congratulations, you have finished the installation! For some use cases you may still 
need to install additional components. Check the description below, as well as the 
:doc:`list of additional configurations <openvino_docs_install_guides_configurations_header>`
to see if your case needs any of them.

This way we refer readers to just one article and make the instruction itself shorter.
We'll try to make further adjustments in the future but aim at retaining the overall structure.

Reply:
Don't you think we explicit step in docs?
E.g.
- download archive / run package managers
- ...
- (optional) configure devices

I'm afraid users can forgot to install GPU / NPU driver if it's not explicitly listed as a **step**